source/funkin/play/PlayState.hx

```diff
@@ -491,9 +491,9 @@ class PlayState extends MusicBeatSubState
   public var camGame:FlxCamera;
 
   /**
-   * The camera which contains, and controls visibility of, a video cutscene.
+   * The camera which contains, and controls visibility of, a video cutscene, dialogue, pause menu and sticker transition.
    */
-  public var camCutscene:FlxCamera;
+  public var camOther:FlxCamera;
```

Comment:
Why replace camCutscene?? That can lead to some broken scripts.
Wouldn't it be smarter to just add a new camera for this?

Reply:
That's true, it also breaks a bunch of cutscene scripts and those would need to be redone.

Reply:
I renamed it because it didn't just handle cutscenes now, but I just reverted it